src/libstore/store-api.cc

```diff
@@ -783,9 +783,16 @@ void copyStorePath(ref<Store> srcStore, ref<Store> dstStore,
 }
 
 
-std::map<StorePath, StorePath> copyPaths(ref<Store> srcStore, ref<Store> dstStore, const StorePathSet & storePaths,
+std::map<StorePath, StorePath> copyPaths(ref<Store> srcStore, ref<Store> dstStore, const RealisedPath::Set & paths,
```

Comment:
I think the original function is basically intact within this? I would prefer the original function stay as in, then, and a new `copyPaths` for the `RealizedPath` stuff parts call the original. Then we maintain a separation of concerns.

Reply:
Oh that's very true. Esp. given that the `StorePathSet` overload is still defined just above. Fixed in b7ce59e